src/clr-icons/shapes/technology-shapes.ts

```diff
@@ -1168,7 +1163,6 @@ export const TechnologyShapes: any = {
   'process-on-vm': ClrShapeProcessOnVM,
 };
 
-Object.defineProperty(TechnologyShapes, 'analytics', descriptorConfig(TechnologyShapes['line-chart']));
```

Comment:
@jeeyun can you make sure we have this `"analytics"` alias in the `"chart-shapes"`? We need to remove the deprecated icons, but we shouldn't remove their aliases. The aliases should go into their new sets too.

Reply:
Yeah I have already added it in this PR! :)

Reply:
oops I missed that sorry.

Reply:
And thanks again working on this :)